fix/fixer_scaleway_access_key.go

```diff
@@ -0,0 +1,50 @@
+package fix
+
+import (
+	"github.com/mitchellh/mapstructure"
+)
+
+// FixerScalewayAccessKey changes the "access_key" of a template
+// to "organization_id".
+type FixerScalewayAccessKey struct{}
+
+func (FixerScalewayAccessKey) Fix(input map[string]interface{}) (map[string]interface{}, error) {
+	// The type we'll decode into; we only care about builders
+	type template struct {
+		Builders []map[string]interface{}
+	}
+
+	// Decode the input into our structure, if we can
+	var tpl template
+	if err := mapstructure.Decode(input, &tpl); err != nil {
+		return nil, err
+	}
+
+	for _, builder := range tpl.Builders {
+		access_keyRaw, ok := builder["access_key"]
```

Comment:
I think you should do some testing here to make sure that the builder type is scaleway; I believe some other builders (alicloud builder, for instance) also use an "access_key" template option.

Reply:
oh geez. Was this here already and I totally missed it?  This is fine.